db/models.py

```diff
@@ -1 +1,42 @@
 from django.db import models
+
+
+class Race(models.Model):
+    RACE_CHOICES = [
+        ("elf", "Elf"),
+        ("dwarf", "Dwarf"),
+        ("human", "Human"),
+        ("ork", "Ork"),
+    ]
+
+    name = models.CharField(max_length=255, unique=True, choices=RACE_CHOICES)
+    description = models.TextField(blank=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
+    race = models.ForeignKey(Race, on_delete=models.CASCADE)
+
+
+class Guild(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(null=True, blank=True)
+
+
+class Player(models.Model):
+    nickname = models.CharField(max_length=255, unique=True)
+    email = models.EmailField(max_length=255, unique=True)
+    bio = models.CharField(max_length=255)
```

Comment:
The `bio` field in the `Player` model is a `CharField` with a max length of 255. If you expect users to write longer biographies, consider using a `TextField` instead.